code/modules/industrial_lift/tram_override_objects.dm

```diff
@@ -24,17 +24,22 @@
 	opacity = FALSE
 
 /obj/machinery/door/window/left/tram
+	icon = 'icons/obj/doors/tramdoor.dmi'
+
 /obj/machinery/door/window/right/tram
+	icon = 'icons/obj/doors/tramdoor.dmi'
 
 /obj/machinery/door/window/left/tram/Initialize(mapload, set_dir, unres_sides)
 	. = ..()
 	RemoveElement(/datum/element/atmos_sensitive, mapload)
 	associated_lift = MAIN_STATION_TRAM
+	INVOKE_ASYNC(src, /obj/machinery/door/window.proc/open)
```

Comment:
just for the record you do not need /obj/machinery/door/window here. the . will do a lookup for the open proc for you.

Reply:
lookup's done on the type of the proc the path is generated in, and done at compile time. This same change applies to most of the uses of proc pathing I've seen here

Reply:
Done.